packages/mobile/scripts/run_app.sh

```diff
@@ -0,0 +1,123 @@
+case "${unameOut}" in
+    Linux*)     MACHINE=Linux;;
+    Darwin*)    MACHINE=Mac;;
+    CYGWIN*)    MACHINE=Cygwin;;
+    MINGW*)     MACHINE=MinGw;;
+    *)          MACHINE="UNKNOWN:${unameOut}"
+esac
+echo "Machine type: $MACHINE"
+echo "Current directory: `pwd`"
+
+# Read values from the .env file and put them in env vars
+ENV_FILENAME="${ENVFILE:-.env}"
+export $(grep -v '^#' $ENV_FILENAME | xargs)
+
+if [ -z "$NETWORK" ]; then
+  echo "No network set."
+  read -p "Use $DEFAULT_TESTNET network set in .env file (y/n)? "
+  if [[ $REPLY =~ ^[Yy]$ ]]; then
+    NETWORK=$DEFAULT_TESTNET
+  else 
+    echo "No network chosen. Exiting."
+    exit 1
+  fi
+fi
+
+# Set DEFAULT_TESTNET in .env file
+sed -i.bak "s/DEFAULT_TESTNET=.*/DEFAULT_TESTNET=$NETWORK/g" $ENV_FILENAME
+
+# Set Hot Reload (saved nav state) in .env file
+sed -i.bak "s/DEV_RESTORE_NAV_STATE_ON_RELOAD=.*/DEV_RESTORE_NAV_STATE_ON_RELOAD=$HOT_RELOAD/g" $ENV_FILENAME
+
+# Set Firebase settings in google service config files
+ANDROID_GSERVICES_PATH="./android/app/src/debug/google-services.json"
+IOS_GSERVICES_PATH="./ios/GoogleService-Info.plist"
+sed -i.bak "s/celo-org-mobile-.*firebaseio.com/celo-org-mobile-$NETWORK.firebaseio.com/g" $ANDROID_GSERVICES_PATH
+sed -i.bak "s/celo-org-mobile-.*firebaseio.com/celo-org-mobile-$NETWORK.firebaseio.com/g" $IOS_GSERVICES_PATH
+
+# Cleanup artifacts from in-place sed replacement on BSD based systems (macOS)
+rm -f $ENV_FILENAME.bak
+rm -f $ANDROID_GSERVICES_PATH.bak
+rm -f $IOS_GSERVICES_PATH.bak
+
+
+# Build Wallet Kit for env
+if [ "$FAST" = false ]; then
+  echo "Building sdk for testnet $NETWORK"
+  yarn build:sdk $NETWORK
+  echo "Done building sdk"
+fi
+
+# Build the app and run it
+if [ $PLATFORM = "android" ]; then
+  echo "Using platform android"
+
+  NUM_DEVICES=`adb devices -l | wc -l`
+  if [ $NUM_DEVICES -lt 3 ]; then 
+    echo "No android devices found"
+    exit 1
+  fi
+
+  # Run jettify to fix non-android-x compatible libs
+  if [ "$FAST" = false ]; then
+    echo "Jetifying react native libraries"
+    cd ../../ && yarn run jetify && cd packages/mobile
+    echo "Jetified"
+  fi
+
+  if [ $MACHINE = "Mac" ]; then
+    echo "Starting packager in new terminal"
+    RN_START_CMD="cd `pwd`;yarn react-native start"
+    OSASCRIPT_CMD="tell application \"Terminal\" to do script \"$RN_START_CMD\""
+    echo "FULLCMD: $OSASCRIPT_CMD"
+    osascript -e "$OSASCRIPT_CMD"
+    # Run android without packager because RN cli doesn't work with yarn workspaces
+    yarn react-native run-android --appIdSuffix \"debug\" --no-packager
+  else 
+    # Run android without packager because RN cli doesn't work with yarn workspaces
+    yarn react-native run-android --appIdSuffix \"debug\" --no-packager
+    yarn react-native start 
+  fi
+
+elif [ $PLATFORM = "ios" ]; then
+  echo "Using platform ios"
+  # TODO have iOS build and start from command line
+  echo -e "\nFor now ios must be build and run from xcode\nStarting RN bundler\n"
```

Comment:
We can do `yarn react-native run-ios --simulator "iPhone 11"`. Last time I checked it worked.

Reply:
For me it didn't work but I'll try again :)

Do you want it to start it though? I thought you preferred Xcode? Maybe there's a way to trigger Xcode actions from command line?

Reply:
oh yeah, iPhone 11 only works from Xcode 11, 
I just checked, and `yarn react-native run-ios --simulator "iPhone Xʀ"` works.

I think it makes sense to build it from the command line by default and have another switch to open Xcode too. What do you think?